docs/operators-and-hooks-ref.rst

```diff
@@ -671,6 +619,54 @@ These integrations allow you to perform various operations within the Google Clo
      - :mod:`airflow.providers.google.cloud.operators.vision`
      -
 
+   * - `Compute Engine <https://cloud.google.com/compute/>`__
```

Comment:
https://github.com/apache/airflow/pull/6487/files#diff-471d3adab9e55d525a899f46345f3b2fL501
It seems to me that order was introduced here.

Reply:
Sorry, I am afraid I don't quite understand what you mean.

Reply:
Sorry, I am afraid I don't quite understand what you mean.

Reply:
We use alphabetical sorting here, but ignoring the "cloud" prefix.
<img width="1377" alt="Screenshot 2019-11-01 at 17 26 30" src="https://user-images.githubusercontent.com/12058428/68039598-c6c77880-fccc-11e9-86b5-89c06727ad1d.png">

Reply:
Are we sort by hook or operator name rather than 'display' name?

Reply:
By display name, but we ignore the prefix "Cloud" . This prefix has no added value because its use by Google is not uniform and not ordered.
For example:
When we have services: Compute Engine, Cloud SQL,  Cloud Storage, PubSub.
The classic alphabetical order creates the following list.
* Cloud SQL
* Cloud Storage
* Compute Engine
* PubSub
* Spanner
However, we want to ignore the Cloud prefix, so the list will look like this:
* Compute Engine 
* PubSub
* Spanner 
* SQL=> Cloud SQL
* Storage => Cloud Storage
This will allow you to easily find services, because some services are not placed at the very end of the list when Google added a prefix, which is not consistently used even by Google.

Reply:
I know the Google part, I want to know if others are do the same things? like Apace part also ignore `Apache` prefix and then order.

Reply:
That sounds sensible to me, but I haven't focused on it.

I wonder more how to sort transfer operators. Should we sort by source or destination?it would be perfect for the user to be able to choose, but I left it to be done at a later time, because first I want to make a new look for all documentation - AIP-11.

Reply:
Ok, so we would not accept this PR until AIP-11? right?

Reply:
No. We can accept changes in the content, but I'd rather limit the changes in the Sphinx template.

Reply:
OK, I will close the PR

Reply:
Please no. I want accept this PR.

Reply:
I think you want to change sphinx template to sorted the content? Am I wrong?

Reply:
The script will provide an additional option, but by default the tables should also be sorted